web/themes/custom/nysenate_theme/src/patterns/components/nysenate-featured-event/nysenate-featured-event.twig

```diff
@@ -26,12 +28,16 @@
   {% if street_1 or street_2 %}
     <div class="c-event-address">{% if street_1 %}{{ street_1 }}<br>{% endif %}{{ street_2 }}</div>
   {% endif %}
-
-</div>
-<div class="c-event-time">
+  <div class="c-event-time">
     <span class="date-display-single">
       <span class="date-display-range">
-        <span class="date-display-start">{{ time_start }}</span> to <span class="date-display-end">{{ time_end }}</span>
+        <span class="date-display-start">{{ time_start }}</span>
+        {% if time_end != time_start %}
```

Comment:
I saw one of the patches allowed for end time to be optional.  Maybe change this:
` if time_end > time_start `

Reply:
i would if i could figure out a way to pass along the variable ... but this is a view in which they are using the top level views-view template, so there is no easy way to pass along the field variable, this is one of their preprocess gems ... which i can't wait to talk to Aten about. I was able to use the patch on the node page for now, but hope to expand it's use once i have a better understanding of how to refactor these patterns/templates

Reply:
The point here is that if `time_end` is optional, it could be empty.  Assuming `time_start` is populated, then they will not be equal, and line 35 will evaluate `TRUE`.  You can mitigate this by changing the logic to a more narrow comparison (ending time is after starting time).

Reply:
i made the change, i wasn't disputing your point, just saying why i couldn't use the patch yet on this template